.jenkins/pytorch/test.sh

```diff
@@ -401,9 +412,18 @@ else
   test_benchmarks
   test_rpc
   if [[ "$BUILD_ENVIRONMENT" == *coverage* ]]; then
+    # code coverage for python sources
     pushd test
-    echo "Generating XML coverage report"
+    echo "Generating XML coverage report for python sources"
     time python -mcoverage xml
     popd
+
+    # code coverage for c++ sources
+    pushd build
+    echo "Generating lcov coverage report for c++ sources"
+    lcov --capture --directory . --output-file coverage.info
```

Comment:
Please add `time` to report how long it takes to run this command
```suggestion
    time lcov --capture --directory . --output-file coverage.info
```